nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/SelectHiveQL.java

```diff
@@ -113,6 +126,17 @@
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
 
+    public static final PropertyDescriptor HIVEQL_POST_QUERY = new PropertyDescriptor.Builder()
+            .name("hive-post-query")
+            .displayName("HiveQL Post-Query")
+            .description("HiveQL post-query to execute. Semicolon-delimited list of queries. "
+                    + "Example: 'set tez.queue.name=default; set hive.exec.orc.split.strategy=HYBRID; set hive.exec.reducers.bytes.per.reducer=258998272'. "
```

Comment:
I'm clearly not a Hive expert but does it make sense to give this kind of examples for post-queries?

Reply:
In my opinion - there shouldn't be "post" queries at all, as after "select" it doesn't make sense to run anything. But based on discussion, there is a demand for "post-queries", that's why we implement.  I'll be happy to get any suggestion on what could be a good example for post-query :)